snapcraft/tests/test_tour.py

```diff
@@ -51,11 +49,13 @@ class TestScaffoldExample(TestCase):
     def setUp(self):
         super().setUp()
         self.temp_dir = None
+        self.cwd = os.getcwd()
 
     def tearDown(self):
         with suppress(OSError):
             if self.temp_dir:
                 shutil.rmtree(self.temp_dir)
+        os.chdir(self.cwd)
```

Comment:
Oh, I didn't see that base class add that fixture, indeed, using this and preventing creating other temp directory when that makes sense for the tests.